### ISSUE TITLE
Generic ESP8285 Full-duplex 2.4GHz RX as TX  - no WIFI ?

### ISSUE THREAD

**Issue by sisari** [ExpressLRS/ExpressLRS#2874]
I have accidentally flashed this firmware into the ESP8285 of this ELRS controller, via WIFI:
https://www.mateksys.com/?portfolio=f405-wte#tab-id-2

Now onboard RX is dead and moreover I cant see any SSID broadcasted on WIFI ...!?
Any suggestions ? flash ESP8285  via UART with eptool  ? 
Happilly I saved the original firmware in the  ESP8285  named: DIY_2400_RX_ESP8285_SX1280_3.0.1 (2470fc).bin

**Comment by pkendall64** [comment]
Yes you will need to reflash using a UART adapter and esptool. Or use configurator and choose the happymodel EP1 target.
